nmtwizard/preprocess/prepoperator.py

```diff
@@ -261,108 +257,3 @@ def _preprocess_tu(self, tu, meta_batch):
             if (c(tu)):
                 return []
         return [tu]
```

Comment:
Shouldn't we separate Pipeline and Operator (base classes) to different files as well, to make everything crystal-clean ? :)

Originally Pipeline was a type of operator, that's why they ended up in the same file.

Reply:
That's right. I can do a separate PR for that.

Reply:
OK, let's merge this one.